src/late_multicellular_stage/MulticellularConvolutionDispayer.cs

```diff
@@ -82,7 +81,7 @@ protected override void Dispose(bool disposing)
     {
         if (disposing)
         {
-            material?.Dispose();
+            // ...
```

Comment:
This dispose method likely ends up empty now and it can be just entirely removed.